LFGBulletinBoard/RequestList.lua

```diff
@@ -691,11 +691,9 @@ function GBB.GetDungeons(msg,name)
 		if nameLevel>0 and nameLevel<40 then
 			dungeons["DM"]=true
 			dungeons["DM2"]=false
-			dungeons["DFC"]=false
 		else
 			dungeons["DM"]=false
 			dungeons["DM2"]=true
-			dungeons["DFC"]=true
```

Comment:
This is causing errors outside of SoD, and Diremaul listings to organized under DFC in SoD

Reply:
half the shit has been so long forgot what it was there for